src/coreclr/jit/lower.cpp

```diff
@@ -4313,6 +4313,14 @@ bool Lowering::TryLowerConditionToFlagsNode(GenTree* parent, GenTree* condition,
     DISPTREERANGE(BlockRange(), condition);
     JITDUMP("\n");
 
+#ifdef DEBUG
+    if (comp->compStressCompile(Compiler::STRESS_SKIP_COND_NODE_LOWERING, 50))
+    {
+        JITDUMP("JitStress: skip lowering attempt\n");
+        return false;
+    }
+#endif // DEBUG
```

Comment:
50% of methods seems a bit high given the frequency of conditional branches.. maybe something like 10% is sufficient?

Nit: the `ifdef` is unnecessary, there is a release version of `compStressCompile` that always returns false (but feel free to leave it if you prefer to make it more visible that this is debug-only).

Reply:
Sounds good, I'll update it.